ska_conda/pkg_defs/ska3-template/bin/flt_envs

```diff
@@ -0,0 +1,79 @@
+#!%{PREFIX}%/bin/python -E
+
+"""
+Print to stdout the appropriate shell commands to initialize the standalone
+Ska environment.
+
+Usage::
+
+  usage: flt_envs [-h] [--shell SHELL]
+
+  Set flight environment vars
+
+  optional arguments:
+    --shell SHELL  Shell name (csh|tcsh|bash|sh|ksh) default=bash
+"""
+
+import os
+from os.path import dirname, abspath
+import sys
+
+
+SHELL = 'bash'
+for idx, arg in enumerate(sys.argv):
+    if arg == '-shell':
+        SHELL = sys.argv[idx + 1]
+if SHELL not in ['csh', 'tcsh', 'bash', 'sh', 'ksh']:
+    raise ValueError("shell '%s; not supported" % SHELL)
+
+
+# Use the convention that flt_envs must be in $SKA/bin to derive SKA and
+# OVERRIDE any existing value.
+ska = dirname(dirname(abspath(__file__)))  # from $SKA/bin/flt_envs to $SKA
+
+# Set environment vars
+envs = dict(('SKA_' + suffix.upper(), os.path.join(ska, suffix))
+        for suffix in ('arch', 'data', 'lib', 'share', 'bin', 'idl'))
```

Comment:
`SKA_ARCH_OS` is definitely needed and should be the same as `SKA`.

Reply:
Is it though?  Did pretty well on the pass through ska_testr tests without it and *not* setting it here might be a good way to flush out other implicit dependencies.

Reply:
I'm also not sure if we need to be using that loop just above here that is apparently setting SKA_ARCH and SKA_IDL.

Reply:
Right now the `acis_taco` cron job will definitely fail:
https://github.com/search?q=org%3Asot+ska_arch_os&type=Code

There are enough other references that I'd rather just leave this in for now and clean up later.

Reply:
I originally was going to say remove `arch` and `idl` but have been trying (for your sake) to let go of minor nits that can be cleaned later.  I had a number of other comments that also went unsaid...  But we do still want SKA_DATA (and lib, share, bin).

Reply:
That's fine.  I was just first shoot for "passes ska_testr" tests.  I think we should probably expand the tests to have more coverage.  

Also, starcheck is not PY3-working code yet, so there would be a bunch of changes to that project to work with this new env anyway.

Reply:
We are in the realm where this is basically close and by far the most efficient path for finding problems is to wrap it up and start using it.  This does not have to be perfect in version 0.1 (i.e. we can leave some non-functioning vars and cruft) and we don't have to think through every possible problem right now.